### PR TITLE
Extend statistics with unreleased count

### PR DESCRIPTION
**Pull request by polesye**

### CODE REVIEW

File: moodler/moodle.py [hunk 1]
```diff
@@ -67,23 +82,22 @@ def submissions_statistics(course_id, is_verbose=False, download_folder=None):
...
+        submitted_submissions = assignment.submitted()
+        ungraded_submissions = assignment.ungraded()
+        ungraded_amount = len(ungraded_submissions)
+        resubmissions_amount = len([s for s in submitted_submissions if s.resubmitted])
+        unreleased_amount = len([s for s in submitted_submissions if s.released])
         ungraded_ignored = []
 
-        for submission in current_assignment_ungraded:
-            if submission.resubmitted:
-                current_assignment_resubmissions_amount += 1
```
Comment (polesye) [inline moodler/moodle.py:85]: Previously, the logic counted the number of resubmissions for the ungraded submissions only.
It means the number may decrease between runs. Could you confirm that?

File: moodler/moodle.py [hunk 2]
```diff
@@ -67,23 +82,25 @@ def submissions_statistics(course_id, is_verbose=False, download_folder=None):
...
     assignments = get_assignments(course_id)
     users_map = get_students(course_id)
 
     for assignment in assignments:
-        current_assignment_submissions_amount = len(assignment.submissions)
-        current_assignment_ungraded = assignment.ungraded()
-        current_assignment_ungraded_amount = len(current_assignment_ungraded)
-
-        current_assignment_resubmissions_amount = 0
+        submissions_amount = len(assignment.submissions)
```
Comment (polesye) [inline moodler/moodle.py:94]: do we need to count ALL submissions or submitted only?

Comment (KeshetS) [reply moodler/moodle.py:94]: We need all submissions. This way we can easily see if all the students submitted the exercise at least once, or if there is an exercise that the students have difficulty with.

File: moodler/moodle.py (hunk 1 above)
Comment (KeshetS) [reply moodler/moodle.py:85]: Counting the number of resubmissions for the ungraded submissions only is what we need here. We use this to prioritize which exercises to check first. So if we see that all submissions are just resubmissions, we might prioritize new submissions in other exercises first.

File: moodler/moodle.py [hunk 3]
```diff
@@ -67,23 +84,30 @@ def submissions_statistics(course_id, is_verbose=False, download_folder=None):
...
+        submitted_submissions = assignment.submitted()
+        ungraded_submissions = assignment.ungraded()
+        ungraded_amount = len(ungraded_submissions)
+        # Resubmission
+        resubmissions_amount = len([s for s in ungraded_submissions if s.resubmitted])
+        # Graded but unreleased
+        unreleased_amount = len(
```
Comment (polesye) [inline moodler/moodle.py:101]: @KeshetS fyi^^

**Pull request merged**